modyn/evaluator/internal/metrics/roc_auc.py

```diff
@@ -33,7 +33,10 @@ def _dataset_evaluated_callback(self, y_true: torch.Tensor, y_pred: torch.Tensor
         desc_score_indices = torch.argsort(y_pred, descending=True)
         y_score = y_pred[desc_score_indices]
```

Comment:
@XianzheMa  Is this related?
![image](https://github.com/user-attachments/assets/0c5d4730-e97d-449a-bdf5-22c13cc03b03)

Reply:
hmmm I don't think so. What dataset/model did you use in this experiment?

Reply:
I think we solved it now. Seems to be related with an invalid choice of evaluation transformer function